definitions/contested/json/CaseEventToFields/DFR-3516-nonprod.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
Does this have to go into CaseEventToFields.json? It seems like it needs to go into prod when merged

Reply:
Good spot.

Reply:
3699b5d71d191f9e92b1813631aec130cc1fef02